src/filecoin/FilecoinEngine.ts

```diff
@@ -331,6 +331,8 @@ export class FilecoinEngine extends CurrencyEngine<
     }): void => {
       this.addTransaction(this.currencyInfo.currencyCode, tx)
       this.onUpdateTransactions()
+      this.otherData.knownTxids[tx.txid] = true
```

Comment:
There is already an `engine.txIdMap` you should use. It's automatically kept up  to date by `addTransaction`

Reply:
Is it available after loading the engine? If so, I'll use that instead of the `otherData` which does get loaded.